# HG changeset patch
# User Muhali <muhali@shaw.ca>
# Date 1343728681 -7200
#      Tue Jul 31 11:58:01 2012 +0200
# Node ID 3691a606283e2a64e941f28b9c00aa23f931eb5e
# Parent  46b19589b5934cd6e5a2e17bef1661ceae1e4a95
Fix typo in pkg.m reported in bug #36994.

* pkg.m: fix typo in call to canonicalize_file_name.

diff --git a/scripts/pkg/pkg.m b/scripts/pkg/pkg.m
--- a/scripts/pkg/pkg.m
+++ b/scripts/pkg/pkg.m
@@ -423,17 +423,17 @@ function [local_packages, global_package
         prefix = files{1};
         if (! exist (prefix, "dir"))
           [status, msg, msgid] = mkdir (prefix);
           if (status == 0)
             error("cannot create prefix %s: %s", prefix, msg);
           endif
           warning ("creating the directory %s\n", prefix);
         endif
-        local_packages = prefix = canonicalize_filename (prefix);
+        local_packages = prefix = canonicalize_file_name (prefix);
         user_prefix = true;
         if (length (files) >= 2 && ischar (files{2}))
           archprefix = files{2};
           if (! exist (archprefix, "dir"))
             [status, msg, msgid] = mkdir (archprefix);
             if (status == 0)
               error("cannot create archprefix %s: %s", archprefix, msg);
             endif
